src/posts/2020-02-23-apprenticeship-and-summer-camp-programs/index.md

```diff
@@ -4,6 +4,7 @@ title: Apprenticeship and Summer Camp programs
 author: laura-esteves
 date: 2020-02-23
 cover: ./cover.png
+seoImage: ./cover.png
```

Comment:
Why specify `seoImage` when its value is the same as `cover` if we're using the cover automatically when this is not specified?

Reply:
the build was failing, because there was no post at all with this field set at the moment, so gatsby couldn't infer the type.
I just fixed by using the new field once, in a backward-compatible way